vendor/github.com/stretchr/testify/require/doc.go

```diff
@@ -0,0 +1,28 @@
+// Package require implements the same assertions as the `assert` package but
```

Comment:
Thanks, I just noticed this missing dependency too and it will be added in v0.6.0 https://github.com/percona/mongodb_exporter/pull/103